doc/source/contribute.rst

```diff
@@ -68,11 +68,13 @@ At this point, wait for reviewer approval and conclude any discussion that arise
 Release checklist
 ~~~~~~~~~~~~~~~~~
 
-- [ ] Check that the [build is passing](https://travis-ci.org/kivy/python-for-android)
+- [ ] Check that the builds are passing
+  - [ ] [Travis build](https://travis-ci.org/kivy/python-for-android)
+  - [ ] [GitHub Action](https://github.com/kivy/python-for-android/actions)
 - [ ] Run the tests locally via `tox`: this performs some long-running tests that are skipped on Travis.
 - [ ] Build and run the [on_device_unit_tests](https://github.com/kivy/python-for-android/tree/master/testapps/on_device_unit_tests) app using buildozer. Check that they all pass.
 - [ ] Build and run the following [testapps](https://github.com/kivy/python-for-android/tree/master/testapps) for arch `armeabi-v7a` and `arm64-v8a`:
-  - `python3 setup_testapp_python3_sqlite_openssl.py apk`
+  - [ ] `python3 setup_testapp_python3_sqlite_openssl.py apk`
     - [ ] `armeabi-v7a`
     - [ ] `arm64-v8a`
 - [ ] Check that the version number is correct
```

Comment:
I would modify these lines, since, IMHO, the `on_device_unit_tests` app has the same functionality than `testapp_python3_sqlite_openssl` plus a little more (we can check device orientation,if service is working, kivy's keyboard optionts...). So, I would suggest to test `on_device_unit_tests`, because we also could avoid to do the local build (since we have it done in an isolated system with github actions). Still, I would put the command needed to perform a local build for a given arch, just in case that github actions fails or who knows...

Also I would suggest to put the command relative to the project (not inside `testapps`) since I think that, if you take an programming IDE like pycharm, the default terminal path is at project level, so it will make easier to copy/paste the command and run it from inside an IDE without entering the `testapps` direcctory. Also I think that is more explicit.

```
- [ ] Build (or download from github actions) and run the following [testapps](https://github.com/kivy/python-for-android/tree/master/testapps/on_device_unit_tests) for arch `armeabi-v7a` and `arm64-v8a`:
  - [ ] on_device_unit_tests
    - [ ] `armeabi-v7a` (cmd: `python3 testapps/on_device_unit_tests/setup.py  --arch=armeabi-v7a --debug`)
    - [ ] `arm64-v8a` (cmd: `python3 testapps/on_device_unit_tests/setup.py  --arch=arm64-v8a --debug`)
```

Anyhow,  I would suggest to have @inclement approval for the suggested changes I made (he has far better vision than mine and this is very sensitive stuff...), for your changes, no problem at all,so you have my approval.

**Thanks** :smile: